test.js

```diff
@@ -3,7 +3,7 @@ import path from 'path';
 import { createRequire } from 'module';
 import tmp from 'tmp';
 import test from 'ava';
-import { factory, runTasks } from 'release-it/test/util/index.js';
+import { factory, runTasks } from 'release-it/test/util';
```

Comment:
The existing import should still work. Why was this change necessary? (release-it's exports contain `test/util/index.js`)

Reply:
Oh I see what's happened. `release-it`'s exports alias the path, but that's not a valid path for esm. I'll see if I can get it to work.

Reply:
So the issue that remains is that you can't use directory imports in ESM, but the full path is not exposed via `release-it`'s exports field. I think this issue may need to be resolved in `release-it` itself.

Reply:
So it looks like changing the `exports` field within `release-it` will fix the issue. I'll open a PR there.

Reply:
Opened an issue (https://github.com/release-it/release-it/issues/920).

Reply:
That issue is resolved and a new verison released. I'll update this PR accordingly.

Reply:
Aaaaand the fix didn't work as the path aliased is incorrect. I've opened a [new PR](https://github.com/release-it/release-it/pull/921) in `release-it` to fix the path.

Reply:
I've worked around this issue for now, somewhat heavy handedly for this PR, by converting the test harness to `jest` over `ava`. This also serves the purpose of matching the test infra with the sibling `release-it-yarn-workspaces` plugin. Most importantly, it unblocks shipping this package.